workspaces/libnpmexec/package.json

```diff
@@ -67,6 +67,7 @@
     "npm-package-arg": "^12.0.0",
     "pacote": "^21.0.0",
     "proc-log": "^5.0.0",
+    "proper-lockfile": "^4.1.2",
```

Comment:
Note: If we keep this, when I review this I will be branching off of this PR and isolating this deps change in its own commit, authored by me.  We are extremely cautious surrounding deps submissions, and even moreso in npm itself since they're bundled.  (cf https://github.com/npm/cli/blob/latest/CONTRIBUTING.md#dependencies)